src/layman/upgrade/upgrade_v2_0.py

```diff
@@ -267,18 +360,64 @@ def migrate_layers():
         else:
             logger.warning("      geoserver.sld already exists!")
 
+        # Move thumbnail file
+        logger.info("      moving thumbnail file")
+        src_thumbnail_path = f"{settings.LAYMAN_DATA_DIR}/workspaces/{workspace}/layers/{layername}/thumbnail/{layername}.png"
+        dst_thumbnail_path = f"{settings.LAYMAN_DATA_DIR}/layers/{layer_uuid}/thumbnail/{layer_uuid}.png"
+        os.makedirs(f"{settings.LAYMAN_DATA_DIR}/layers/{layer_uuid}/thumbnail/", exist_ok=True)
+        shutil.move(src_thumbnail_path, dst_thumbnail_path)
+
```

Comment:
I suggest to carefully delete the original layer directory with `os.rmdir`.

Reply:
Incorporated.